ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -95,6 +95,15 @@ export SERVICE_BRANCH={{ PULL_PULL_SHA }}
 export OPENSHIFT_INSTALL_RELEASE_IMAGE={{ RELEASE_IMAGE_LATEST }}
 {% endif %}
 
+{% if PLATFORM == "vsphere" %}
+export PLATFORM=vsphere
+export VIP_DHCP_ALLOCATION=true
```

Comment:
Same as above
Less defaults here, more on test-infra

Reply:
this one handled?

Reply:
test-infra uses DHCP_ALLOCATION for vsphere by default but CI env doesn't support it so disabling it here